src/redismodule.h

```diff
@@ -158,15 +158,16 @@ This flag should not be used directly by the module.
 #define REDISMODULE_NOTIFY_EXPIRED (1<<8)     /* x */
 #define REDISMODULE_NOTIFY_EVICTED (1<<9)     /* e */
 #define REDISMODULE_NOTIFY_STREAM (1<<10)     /* t */
-#define REDISMODULE_NOTIFY_KEY_MISS (1<<11)   /* m (Note: This one is excluded from REDISMODULE_NOTIFY_ALL on purpose) */
-#define REDISMODULE_NOTIFY_LOADED (1<<12)     /* module only key space notification, indicate a key loaded from rdb */
+#define REDISMODULE_NOTIFY_MODULE (1<<11)     /* d, module key space notification */
+#define REDISMODULE_NOTIFY_KEY_MISS (1<<12)   /* m (Note: This one is excluded from REDISMODULE_NOTIFY_ALL on purpose) */
+#define REDISMODULE_NOTIFY_LOADED (1<<13)     /* module only key space notification, indicate a key loaded from rdb */
```

Comment:
unfortunately, this would break the ABI and all existing modules, we can't change the order of these bits.